vsintegration/Vsix/VisualFSharpFull/VisualFSharpFull.csproj

```diff
@@ -95,7 +95,7 @@
       <NgenArchitecture>All</NgenArchitecture>
       <NgenPriority>2</NgenPriority>
       <Private>True</Private>
-      <AdditionalProperties>TargetFramework=net472</AdditionalProperties>
+      <AdditionalProperties>TargetFramework=netstandard2.0</AdditionalProperties>
```

Comment:
Change `Microsoft.DotNet.DependencyManager.fsproj` to `<TargetFramework>` (instead of the plural `<TargetFrameworks>`) then you can drop this line since that's the only TFM that project supports.

Reply:
So ... I can try, but every time we've tried to go from TargetFrameworks to TargetFramework we end up with wierd behaviour.